### PR TITLE
Adds deploy docs

### LINKED ISSUE #454: New Pod Owners
Currently, I'm the only one who deploys Moya releases. This sucks because I'm not always available to do it, and I don't want to be a bottleneck for anyone else. 

So, I'm looking for existing contributors who are willing and able to occasionally release a new Moya version. It's not a huge job at all, but I feel like it's a bigger responsibility than being a contributor, so there's absolutely no pressure to accept. @petester42 @colinta @esttorhe @aschuch anyone interested?

(I would need to add some documentation around _how_ to make a release as well.)

### PR DESCRIPTION
**Pull request by ashfurrow**
With new pod owners, I want to document the automated release process.

@colinta @petester42 @sunshinejr @esttorhe Any questions?

Fixes #454.

### CODE REVIEW

**Comment by esttorhe** [comment]
Not from my end; seems pretty straight forward

**Comment by petester42** [comment]
Looks good. Pretty straightforward.

File: docs/Releasing.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Note: to make a release, you need at least one entry in the `Next` section of the changelog.
+
+To make a release:
+
+1. Pull latest from master and make sure your git is clean (the script will fail if it's not).
+1. Run `rake release["X.Y.Z"]`.
+1. Grab a :tea: or :coffee:.
+1. Make sure everything went smoothly.
+
+If anything goes wrong, don't panic! Get in touch with someone else who has released, or [Ash](mailto:ashfurrow.com).
```
Comment (colinta) [inline docs/Releasing.md]: `mailto:ash@ashfurrow.com`?  `me@ashfurrow.com`? :smiley:

Comment (ashfurrow) [inline docs/Releasing.md]: ah, oops >.<

**Comment by esttorhe** [comment]
> /Users/travis/build/Moya/Moya/Demo/Pods/Nimble/Nimble/Adapters/NimbleXCTestHandler.swift:9:64: cannot convert value of type 'String' to expected argument type 'StaticString'

Haven't checked the `.travis.yml` but are we building against/using `Swift 2.2`?
Or the `Podfile` points to `Nimble` with `Swift 2.2` ?

**Comment by ashfurrow** [comment]
The CI failure? Trying to fix that here: https://github.com/Moya/Moya/pull/452

**Comment by esttorhe** [comment]
ahhh; thanks @ashfurrow 
sorry for maybe bringing old stuff up; have been kind of away from the project due to a massive amount of work and trying to catch up on stuff

:grimacing:

**Comment by ashfurrow** [comment]
No worries! April is really busy for me, and I wanted to make sure Moya would be OK if I wasn't available.

**Pull request merged**